File: 프로그래머스/0/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.cs
using System;

public class Solution {
    public string solution(string my_string, string alp) {
        string answer = "";
        char[] arr = alp.ToCharArray();
        for(int i = 0; i< my_string.Length;i++)
        {
            if(my_string[i] == arr[0])
            {
                answer += (char)(my_string[i] - 32);
                continue;
            }
            answer += my_string[i];
        }
        return answer;
    }
}

File: 프로그래머스/0/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <cctype>
using namespace std;

string solution(string my_string, string alp) {
    string answer = my_string;
    replace(answer.begin(), answer.end(), alp[0], static_cast<char>(::toupper(alp[0])));
    
    return answer;
}